android/src/main/java/com/rnwhisper/WhisperContext.java

```diff
@@ -111,6 +187,11 @@ public int startRealtimeTranscribe(int jobId, ReadableMap options) {
 
     isUseSlices = audioSliceSec < audioSec;
 
+    String audioOutputPath = options.hasKey("audioOutputPath")
+        ? options.getString("audioOutputPath").endsWith(".wav") ? options.getString("audioOutputPath")
+            : options.getString("audioOutputPath") + ".wav"
```

Comment:
I think the file extension shouldn't be enforced, we can keep the option of adding or not.